script/update-external-binaries.py

```diff
@@ -8,38 +9,48 @@
 from lib.util import add_exec_bit, download, extract_zip, rm_rf, \
                      safe_mkdir, tempdir
 
-VERSION = 'v1.4.0'
 SOURCE_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
-FRAMEWORKS_URL = 'http://github.com/electron/electron-frameworks/releases' \
-                 '/download/' + VERSION
 
+def parse_args():
+  parser = argparse.ArgumentParser(
+      description='Download binaries for Electron build')
+
+  parser.add_argument('-u', '--root-url', required=True,
+                      help="Root URL for all downloads.")
+  parser.add_argument('-v', '--version', required=True,
+                      help="Version string, e.g. 'v1.0.0'.")
```

Comment:
would be great to have a link to something listing what versions are available

Reply:
Probably ) Honestly I have no idea how those binaries get there =/